src/components/space-panel/components/visited-nodes/visited-nodes.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react"
+import { CollapsibleList, SimpleListItem } from "@rmwc/list"
+import { Text } from "@netdata/netdata-ui"
+import "@material/list/dist/mdc.list.css"
+import "@rmwc/list/collapsible-list.css"
+import "@rmwc/icon/icon.css"
+import { NodesContainer, ListItem, StyledIcon, ListHeaderContainer, NodeUrl } from "./styled"
+import { agentsMock } from "../../mocks"
+
+const Node = ({ agent: { name, validURLs }, visitNode }: any) => {
+  const urls = validURLs ? Object.keys(validURLs) : []
+  return (
+    <CollapsibleList
+      handle={
+        <SimpleListItem
+          text={
+            <>
+              <StyledIcon name="node" />
+              <Text>{name}</Text>
+            </>
+          }
+          metaIcon={urls.length && "chevron_right"}
+        />
+      }
+    >
+      {urls.map((url: string, i: number) => (
+        <ListItem
+          key={i}
+          onClick={() => {
+            visitNode(url)
+          }}
+        >
+          <NodeUrl>{url}</NodeUrl>
+        </ListItem>
+      ))}
+    </CollapsibleList>
+  )
+}
+
+export const VisitedNodes = () => {
+  const agents = agentsMock as any[] // useSelector(selectAgentsSortedList)
+
+  const visitNode = (url: string) => {
+    if (url.includes("http://") || url.includes("https://")) {
+      window.open(url, "_blank")
+    } else {
+      window.open(`http://${url}`, "_blank")
+    }
+  }
+
+  return (
+    <NodesContainer>
+      <CollapsibleList
+        startOpen
+        handle={
+          <ListHeaderContainer>
+            <SimpleListItem metaIcon="chevron_right" text="Visited Nodes" />
```

Comment:
i see a strange issue that clicking this component (or other nodes in the sidebar) is shifting the right-side sidebar upwards/downwards. Do you know something about it? Perhaps it's a old dashboard issue, i'm not sure (if it is, i can look at it later)

Reply:
If you click anywhere else, it will still happen. So it's connected to sidebar.